ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -75,6 +83,20 @@ function component_dpmlfc_node_config_valid = {
   true;
 };
 
+# Validation of dav parameters
+function component_dpmlfc_dav_config_valid = {
+   if ( is_defined(SELF['DiskFlags']) ) {
```

Comment:
You are right, this is a mistake in fact... I need to fix it. Thanks for for noticing.